lib/tokens/index.js

```diff
@@ -8,7 +8,7 @@ const error = require('../error')
 
 module.exports = (log, config) => {
   config = config || {}
-  const lifetimes = config.tokenLifetimes || {
+  const lifetimes = config.tokenLifetimes = config.tokenLifetimes || {
```

Comment:
Fwiw, I tried doing this just now but it caused one of the tests to fail because the value for password-forgot tokens here is different to the default config and the test depends on that. It didn't seem worth changing the test so that it loads config and asserts the value from there, so I've left this as it was. I can still make the change if people want me to though...?